vizro-core/src/vizro/models/_components/card.py

```diff
@@ -55,13 +98,25 @@ def _action_outputs(self) -> dict[str, _IdProperty]:
             "text": f"{self.id}-text.children",
         }
 
+    @staticmethod
+    def _get_value_from_trigger(value: str, trigger: int) -> JsonValue:
+        """Return the given `value` without modification."""
+        return value
+
     @_log_call
     def build(self):
         text = dcc.Markdown(
             id=f"{self.id}-text", children=self.text, dangerously_allow_html=False, className="card-text"
         )
 
-        card_content = (
+        description = self.description.build().children if self.description is not None else [None]
```

Comment:
I guess this would also work just as well. If you agree, let's change it here and in the `container.py` too.
```suggestion
        description = self.description.build().children if self.description else [None]
```